lib/sqlmap/sqlmap_manager.rb

```diff
@@ -8,43 +8,76 @@ def initialize(session)
 
     def new_task
       res = @session.get('/task/new')
-      return JSON.parse(res.body)
+      if res && res.body 
```

Comment:
@Meatballs1 Thanks a lot. Yeah was not thinking clearly. Now the code is much more concise.